src/nl/rubensten/texifyidea/run/compiler/BiberCompiler.kt

```diff
@@ -0,0 +1,35 @@
+package nl.rubensten.texifyidea.run.compiler
+
+import com.intellij.openapi.project.Project
+import nl.rubensten.texifyidea.run.BibtexRunConfiguration
+
+/**
+ * @author Thomas Schouten
+ */
+internal object BiberCompiler : Compiler<BibtexRunConfiguration> {
+
+    override val displayName = "Biber"
+    override val executableName = "biber"
+
+    override fun getCommand(runConfig: BibtexRunConfiguration, project: Project): List<String>? {
+        val command = mutableListOf<String>()
+
+        command.apply {
+            if (runConfig.compilerPath != null) {
+                add(runConfig.compilerPath!!)
+            }
+            else add(executableName)
```

Comment:
```kotlin
add(runConfig.compilerPath ?: executableName)
```

Reply:
Uhm yeah, thanks for noticing. I didn't pay attention when blindly copying from the existing bibtex code :)